test/Microsoft.AspNetCore.Owin.Tests/OwinEnvironmentTests.cs

```diff
@@ -129,6 +130,23 @@ public void OwinEnvironmentSuppliesDefaultsForMissingRequiredEntries()
             Assert.Equal("1.0", env["owin.Version"]);
         }
 
+        [Fact]
+        public void OwinEnvironmentImpelmentsGetEnumerator()
+        {
+            HttpContext context = CreateContext();
+            OwinEnvironment owinEnvironment = new OwinEnvironment(context);
+
+            IEnumerator<KeyValuePair<string, object>> e1 = owinEnvironment.GetEnumerator();
+            Assert.NotNull(e1);
+
+            foreach (var kv in owinEnvironment) { } // This should not throw exception
```

Comment:
These checks are not really useful. It's enough to check that the enumerators are not null.